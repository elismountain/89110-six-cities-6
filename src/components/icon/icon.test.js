import React from 'react';
import {render} from '@testing-library/react';
import Icon from './icon';

describe(`Icon should render correctly`, () => {
  it(`Icon should render correctly without size prop`, () => {
    const {container} = render(
        <Icon />
    );
    expect(container).toMatchSnapshot();
  });

  it(`Icon should render correctly with size prop`, () => {
    const {container} = render(
        <Icon size={20} />
    );
    expect(container).toMatchSnapshot();
  });
});
