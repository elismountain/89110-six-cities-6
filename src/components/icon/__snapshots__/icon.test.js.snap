// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render correctly Icon should render correctly with size prop 1`] = `
<div>
  <svg
    fill="none"
    height="20"
    stroke="#777"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
    viewBox="0 0 24 24"
    width="20"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M9 21H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h4"
    />
    <polyline
      points="16 17 21 12 16 7"
    />
    <line
      x1="21"
      x2="9"
      y1="12"
      y2="12"
    />
  </svg>
</div>
`;

exports[`Icon should render correctly Icon should render correctly without size prop 1`] = `
<div>
  <svg
    fill="none"
    height="24"
    stroke="#777"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
    viewBox="0 0 24 24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M9 21H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h4"
    />
    <polyline
      points="16 17 21 12 16 7"
    />
    <line
      x1="21"
      x2="9"
      y1="12"
      y2="12"
    />
  </svg>
</div>
`;
